uniwersytet-gdanski-kulturoznawstwo-przypis.csl

```diff
@@ -242,167 +185,101 @@
-                  </if>
-                </choose>
+                <text macro="locator"/>
+                <text macro="url-accessed"/>
               </group>
             </else-if>
             <else-if type="song" match="any">
-              <group>
-                <text macro="author" suffix=", "/>
-                <text macro="title" suffix=", "/>
-                <text term="in" prefix="[" suffix=":] "/>
+              <group delimiter=", ">
+                <text macro="author"/>
+                <text macro="title"/>
                 <choose>
-                  <if match="any" variable="composer">
-                    <names variable="composer" suffix=", ">
-                      <name delimiter-precedes-last="always" initialize-with=". "/>
-                      <et-al font-style="italic"/>
-                    </names>
+                  <if match="any" variable="collection-title">
+                    <group delimiter=" ">
+                      <text term="in" prefix="[" suffix=":]"/>
+                      <group delimiter=", ">
+                        <names variable="composer">
+                          <name delimiter-precedes-last="always" initialize-with=". "/>
+                        </names>
+                        <text variable="collection-title" font-style="italic"/>
+                      </group>
+                    </group>
                   </if>
                 </choose>
-                <text variable="collection-title" font-style="italic" suffix=", "/>
                 <text macro="publisher-place-date"/>
-                <choose>
-                  <if match="any" variable="locator">
-                    <text macro="locator" prefix=", "/>
-                  </if>
-                </choose>
+                <text macro="locator"/>
               </group>
             </else-if>
             <else-if type="entry-dictionary entry-encyclopedia" match="any">
               <choose>
                 <if match="any" variable="editor">
-                  <group>
-                    <text macro="author" suffix=", "/>
-                    <text macro="title" suffix=", "/>
-                    <text variable="container-title" font-style="italic" prefix="[hasło w:] " suffix=", "/>
-                    <text macro="editor-translator" suffix=", "/>
-                    <choose>
-                      <if match="any" variable="volume">
-                        <text macro="volume" suffix=", "/>
-                      </if>
-                    </choose>
-                    <choose>
-                      <if match="any" variable="edition">
-                        <text variable="edition" prefix="wyd. " suffix=", "/>
-                      </if>
-                    </choose>
+                  <group delimiter=", ">
+                    <text macro="author"/>
+                    <text macro="title"/>
+                    <text variable="container-title" font-style="italic" prefix="[hasło w:] "/>
```

Comment:
what does the "haslo w" mean? 
We try to use terms/labels where possible and not hardcode text like this.